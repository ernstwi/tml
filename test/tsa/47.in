init a

a -> b -> c
    reset { x }

d -> e -> f

default edge reset { y }

[g, h] -> i -> [j, k]
