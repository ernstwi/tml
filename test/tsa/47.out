{
    "actions": [],
    "clocks": [
        "x",
        "y"
    ],
    "edges": [
        {
            "from": "a",
            "guard": null,
            "reset": [
                "x"
            ],
            "sync": null,
            "to": "b"
        },
        {
            "from": "b",
            "guard": null,
            "reset": [
                "x"
            ],
            "sync": null,
            "to": "c"
        },
        {
            "from": "d",
            "guard": null,
            "reset": null,
            "sync": null,
            "to": "e"
        },
        {
            "from": "e",
            "guard": null,
            "reset": null,
            "sync": null,
            "to": "f"
        },
        {
            "from": "g",
            "guard": null,
            "reset": [
                "y"
            ],
            "sync": null,
            "to": "i"
        },
        {
            "from": "h",
            "guard": null,
            "reset": [
                "y"
            ],
            "sync": null,
            "to": "i"
        },
        {
            "from": "i",
            "guard": null,
            "reset": [
                "y"
            ],
            "sync": null,
            "to": "j"
        },
        {
            "from": "i",
            "guard": null,
            "reset": [
                "y"
            ],
            "sync": null,
            "to": "k"
        }
    ],
    "locations": [
        {
            "id": "a",
            "initial": true,
            "invariant": null
        },
        {
            "id": "b",
            "initial": false,
            "invariant": null
        },
        {
            "id": "c",
            "initial": false,
            "invariant": null
        },
        {
            "id": "d",
            "initial": false,
            "invariant": null
        },
        {
            "id": "e",
            "initial": false,
            "invariant": null
        },
        {
            "id": "f",
            "initial": false,
            "invariant": null
        },
        {
            "id": "g",
            "initial": false,
            "invariant": null
        },
        {
            "id": "h",
            "initial": false,
            "invariant": null
        },
        {
            "id": "i",
            "initial": false,
            "invariant": null
        },
        {
            "id": "j",
            "initial": false,
            "invariant": null
        },
        {
            "id": "k",
            "initial": false,
            "invariant": null
        }
    ]
}
